Resources/Locales/ko/openspades.po

```diff
@@ -1102,13 +1102,13 @@ msgid ""
 "FXAA: Performs antialiasing by smoothing artifacts out as a post-process.|"
 "Off|MSAA 2x|4x|FXAA"
 msgstr ""
-"안티앨리어싱: 계단 현상을 줄여 주는 기술을 활성화합니다.\n"
+"앤티앨리어싱 : 오브젝트의 윤곽에 발생하는 계단 현상을 줄여 줍니다.\n"
 "\n"
-"MSAA: 중간-고해상도의 이미지를 생성함으로써 안티앨리어싱을 수행합니다. 가장 "
-"멋지지만, 몇몇 설정과는 호환되지 않습니다.\n"
+"MSAA : 보통의 경우보다 고해상도의 이미지를 생성하여 앤티앨리어싱을 수행합니"
+"다. 가장 멋져 보이지만, 몇몇 설정과는 호환되지 않습니다.\n"
 "\n"
-"FXAA: 포스트 프로세싱을 통해 물체를 부드럽게 함으로써 안티앨리어싱을 수행합니"
-"다. |끔|MSAA 2x|4x|FXAA"
+"FXAA : 계단 현상을 감지하여 포스트 프로세싱을 통해 안티 앨리어싱을 수행합니"
+"다. | 꺼짐 | MSAA 2x | 4x | FXAA"
```

Comment:
Can you remove the spaces around each item? It needs to be consistent across languages.